baal_dashboard/app.py

```diff
@@ -14,9 +17,63 @@
 
 app = FastAPI()
 
+# Add Options for CORS
+# TODO: Change the options below if you plan to put this into prod
+
+origins = [
+    "*",
+]
+
+app.add_middleware(
+    CORSMiddleware,
+    allow_origins=origins,
+    allow_credentials=True,
+    allow_methods=["*"],
+    allow_headers=["*"],
+)
+ 
 @app.get('/experiments')
-def get_experiments():
-    return [exp.experiment_id for exp in mlflow.MlflowClient().search_experiments()]
+def get_experiments() -> List[Experiment]:
+    '''
+    The function returns data for runs under each experiment ID.
+
+    Args:
+    ----
+    None
+
+    Return:
+    ----
+    exp_details : Returns a List of Experiment and Run Id's
+    
+    '''
+
+    exp_details = []
+
+    experiments = [exp for exp in mlflow.MlflowClient().search_experiments()]
+
+    # Iterate through each experiment
+    for exp in experiments:
+
+        exp_dict = {}
```

Comment:
[Misc] Could be a function that returns an `Experiment` so we can keep adding stuff to the type easily.

Reply:
I have moved the function to create the Experiment data into a separate utils functions. Let me know if this is what you were looking for.